docs/dev/framework/page-controllers.md

```diff
@@ -320,6 +335,90 @@ class ExamplePageController
 ```
 
 
+## URL Generation
+
+Within the database all pages are stored in the `tl_page` table. An entry for a page will be created there when you 
+create a new page for any page type (including your page controllers). Instances of pages in Contao are generally 
+represented by the `Contao\PageModel`. This class allows you to generated URLs to pages via its `getFrontendUrl` and
```

Comment:
represented by the `Contao\PageModel`. This class allows you to generate URLs to pages via its `getFrontendUrl` and